sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/TableIdentifierParserSuite.scala

```diff
@@ -53,10 +55,96 @@ class TableIdentifierParserSuite extends SparkFunSuite {
     "bigint", "binary", "boolean", "current_date", "current_timestamp", "date", "double", "float",
     "int", "smallint", "timestamp", "at", "position", "both", "leading", "trailing", "extract")
 
-  val hiveStrictNonReservedKeyword = Seq("anti", "full", "inner", "left", "semi", "right",
+  val hiveStrictNonReservedKeywords = Seq("anti", "full", "inner", "left", "semi", "right",
```

Comment:
hmm, seems we don't have reserved keywords at all before this PR. We only have non-reserved keywords and strict-non-reserved keywords...